### PR TITLE
ARN-2063 and ARN-2064 - Fix routing logic for Accommodation and update question set

### PR DESCRIPTION
**Pull request by devkokov**

### CODE REVIEW

File: app/sbna-poc/v1_0__initial-form/fields/accommodation.ts [hunk 1]
```diff
@@ -644,7 +647,8 @@ export const noAccommodationFields: Array<FormWizard.Field> = [
     text: 'Why does [subject] have no accommodation?',
     hint: { html: noAccommodationHint, kind: 'html' },
     code: 'no_accommodation_reason',
-    type: FieldType.Radio,
+    type: FieldType.CheckBox,
+    multiple: true,
     validate: [{ type: ValidationType.Required, message: 'Select why they have no accommodation' }],
     options: [
       { text: 'Alcohol related problems', value: 'ALCOHOL_PROBLEMS', kind: 'option' },
```
Comment (Yasmin-jones) [inline app/sbna-poc/v1_0__initial-form/fields/accommodation.ts:654]: I can't see on github but was this also changed, on line 658?
from:
labelClasses: getMediumLabelClassFor(FieldType.Radio), 
To:
labelClasses: getMediumLabelClassFor(FieldType.CheckBox)

**Review by Yasmin-jones — APPROVED** [review]

**Review by LucasCairns — APPROVED** [review]
LGTM 👍

File: app/sbna-poc/v1_0__initial-form/fields/accommodation.ts (hunk 1 above)
Comment (devkokov) [reply app/sbna-poc/v1_0__initial-form/fields/accommodation.ts:654]: well spotted! have updated it now, thanks

**Review by Yasmin-jones — APPROVED** [review]

File: app/sbna-poc/v1_0__initial-form/fields/accommodation.ts (hunk 1 above)
Comment (Yasmin-jones) [reply app/sbna-poc/v1_0__initial-form/fields/accommodation.ts:654]: great :)

**Pull request merged**